.github/CODEOWNERS

```diff
@@ -1,3 +1,12 @@
 # https://help.github.com/en/articles/about-code-owners
 
 /custom/aaa/ @emord
+
+*.html @orangejenny
+*.js @orangejenny
+*.less @orangejenny
```

Comment:
I'm not sure how well this will work either from the docs:

> Order is important; the last matching pattern takes the most precedence

I think as this file is added to, you may get notified less and less

> opens a pull request that only modifies JS files

I'm not sure what happens if someone opened a PR that modified corehq/apps/locations, but also modified python files if that will also notify you (assuming that no one owns locations)

Reply:
Regarding order, yeah, I'm expecting to have to modify this over time. Should probably add a `/.github/` rule.

My read of the docs is that each file in a PR is evaluated separately. I think "only js files" is explaining the interaction between `*` and `*.js`: the global owner doesn't get notified if the PR is only js files, but a mix of js and non-js files would notify both the js owner and the global owner.